src/interfaces/pact-provider-module-options.interface.ts

```diff
@@ -1,7 +1,7 @@
 import { ModuleMetadata, Type } from '@nestjs/common/interfaces';
-import { VerifierOptions } from '@pact-foundation/pact';
+import { VerifierOptions } from '@pact-foundation/pact/src/dsl/verifier/types';
```

Comment:
This should be exposed directly, I think? As in, you should be able to import it from `@pact-foundation/pact`. Or if not, it's a bug in some release of Pact-JS.

Reply:
It is indeed a bug in Pact-JS. I created https://github.com/pact-foundation/pact-js/pull/1009 to fix it.

Reply:
This has been fixed in Pact-JS 10.4.0 - probably the right move is to update the import and set the peer dependency to `^10.4.0`